client/my-sites/earn/memberships/index.jsx

```diff
@@ -345,20 +377,36 @@ class MembershipsSection extends Component {
 				>
 					<h1>{ this.props.translate( 'Confirmation' ) }</h1>
 					<p>
-						{ this.props.translate(
-							'Do you want to disconnect Recurring Payments from your Stripe account?'
-						) }
+						{ isPayments
+							? this.props.translate(
+									'Do you want to disconnect Payments from your Stripe account?'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.